packages/graphql-hooks-memcache/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "lib/graphql-hooks-memcache.js",
   "module": "es/graphql-hooks-memcache.js",
   "unpkg": "dist/graphql-hooks-memcache.min.js",
+  "types": "index.d.ts",
```

Comment:
I *think* you'll have to add this file to the `files` array on line 14

Reply:
Ah yes good spot, I did this in #135.